src/Ng/Catalog2ElfieOptions.cs

```diff
@@ -236,6 +239,50 @@ public static IEnumerable<string> RequiredPackages
             }
         }
 
+        /// <summary>
+        /// The directory which contains text files used to simulate NuGet packages for local assemblies.
+        /// </summary>
+        /// <remarks>To represent the Framework assemblies in the index, we need to create a fake package in
+        /// the index which contains the Framework assemblies/types. The AssemblyPackagesDirectory directory contains the files
+        /// which specify what assemblies to include in the fake package. Each text file is comprised of a list of assemblies.
+        /// These assemblies must exist on the local machine. 
```

Comment:
Yes, the server now requires the assemblies for whatever version of the .NET Framework we want to include. In the one note page describing how to set up a new Azure VM I suggest either installing VS (seems like overkill) or manually copying the reference assemblies from another machine.